DSharpPlus.Lavalink/LavalinkGuildConnection.cs

```diff
@@ -378,5 +492,7 @@ internal Task InternalWebSocketClosedAsync(WebSocketCloseEventArgs e)
             => this._webSocketClosed.InvokeAsync(this, e);
 
         internal event ChannelDisconnectedEventHandler ChannelDisconnected;
```

Comment:
Events, fields, properties and constants alike should be declared at the top of the file. I'm aware this wasn't necessarily your fault though

Reply:
Should I make it public?

Reply:
No, it should be declared at the top of the file before the methods are written